tests/fixtures/scenarios/terraform-sub-components/atmos.yaml

```
@@ -0,0 +1,16 @@
+base_path: "./"
+components:
+  terraform:
+    base_path: "../../components/mock-subcomponents/components/terraform"
```

This seems wrong to me. That we have a `components/terraform` underneath `tests/fixtures/components/terraform/`. It's an unnecessary redundancy. 

```suggestion
    base_path: "../../components/terraform"
```